controllers/AnswerController.js

```diff
@@ -0,0 +1,71 @@
+var flash = require('../utilities/flash.js').flash;
+var Model = require('../models/models.js');
+
+
+module.exports = {
+
+ registerAnswer:  function(req, res){
+  var id_question=req.query.question_id,
+      people_answer=req.body.option,
+      answer = new Model.AnswerModel({
+       id_question:id_question,
+       answer: people_answer
+      });
+  console.log("people anser "+people_answer)
+
+  answer.save(function(error){
+   if(error){
+    flash(req, 'danger', 'registration error!', 'your vote was not registered! try again.');
+   } else {
+    flash(req, 'success', 'vote registered', 'Your vote was registered with success');
+    res.redirect(302,'/answers/showResults/?id='+id_question); 
+   }
+  })
+ }, 
+
+ showResults:  function(req, res){        
+  var idQuest=req.query.id,
+      fullUrl =  req.protocol + '://' + req.get('host') + req.originalUrl,
+      layout = (req.session.user)? 'logged':'main';     
+  count(idQuest, function(err, countData){
+   if (err){console.log(err)}
```

Comment:
You need to send a response to the request, even though there are errors, otherwise the user will never get a response